composer.json

```diff
@@ -15,11 +15,21 @@
             "homepage": "https://github.com/jackalope/jackalope-jackrabbit/contributors"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url":  "https://github.com/richard-fizz/phpcr-utils.git"
+        },
+        {
+            "type": "vcs",
+            "url":  "https://github.com/richard-fizz/phpcr-api-tests.git"
+        }
+    ],
     "require": {
         "php": ">=5.3.3",
         "doctrine/dbal": ">=2.4.5,<3.0.x-dev",
         "phpcr/phpcr": "~2.1.2",
-        "phpcr/phpcr-utils": "^1.2.8",
+        "phpcr/phpcr-utils": "dev-master",
```

Comment:
i merged and tagged this one. please remove the repositories entry for the utils and set this to `^1.2.9`